test/e2e/ipfix_handler.go

```diff
@@ -0,0 +1,171 @@
+package exttest
```

Comment:
copyrights probably needed for new files

Reply:
Fixed